_includes/curation-workflow.md

```diff
@@ -851,7 +851,7 @@ idr0002-screenA-processed.txt
 ![](img/curation-workflow/image9.png)
 
 (see
-[<span class="underline">https://github.com/IDR/idr-metadata/blob/master/idr0002-heriche-condensation/screenA/idr0002-screenA-processed.txt</span>](https://github.com/IDR/idr-metadata/blob/master/idr0002-heriche-condensation/screenA/idr0002-screenA-processed.txt)
+https://github.com/IDR/idr0002-heriche-condensation/blob/master/screenA/idr0002-screenA-processed.txt)
```

Comment:
This link works, but does not render as a link.

![Screen Shot 2020-11-03 at 10 13 25](https://user-images.githubusercontent.com/2478303/97972907-2a432500-1dbd-11eb-8f8d-2c30749a9086.png)

Definite improvement though, not a blocker imho

Reply:
Something's gone wrong.... Markdown is meant to auto-convert links

Reply:
Now fixed

![image](https://user-images.githubusercontent.com/1644105/97982685-0fc47800-1dcc-11eb-96e7-ba848a5c7191.png)

Reply:
Indeed, works fine on my machine too, thanks @manics